DOC-1725: Edit RC Quick Start to reflect new workflow (#2199)
RedisLabs/redislabs-docs: content/rc/subscriptions/create-fixed-subscription.md

Here are few details to keep in mind:

> - You aren't prompted for payment details when creating a Free subscription.

Reviewer: ```suggestion
- You can create a Free subscription without being prompted for payment details.
```
